templates/archive-tsml-ui-classic.php

```diff
@@ -0,0 +1,27 @@
+ <!-- USING TSML UI IN A CLASSIC THEME -->
+<?php
+
```

Comment:
need to either call `tsml_assets();` here so that the theme-specific css loads, or else split that out into its own css file and include that.

to see what i mean, check out the page in twenty fourteen.

without styles
![before](https://user-images.githubusercontent.com/1551689/160255393-ac609fca-5273-4a2d-913c-21dee43437fc.png)

with styles
![after](https://user-images.githubusercontent.com/1551689/160255396-e113942d-c910-4247-9745-b8409a5b7c67.png)

Reply:
Thanks @joshreisner for this tip. It also resolved the problem I was having getting the twentytwentytwo theme to display correctly for both TSML UI and Legacy.